### ISSUE TITLE
propertySet nil

### ISSUE THREAD

**Issue by misuqian** [CosmicMind/Graph#108]
i don't know why . sometime propertySet was nil 

```
internal subscript(name: String) -> AnyObject? {
        get {
            for n in propertySet {
                if let property: ManagedEntityProperty = n as? ManagedEntityProperty{
                    if name == property.name {
                        return property.object
                    }
                }
            }
            return nil
        }
```

**Comment by daniel-jonathan** [comment]
I am looking into this. Thank you for submitting :)

**Comment by daniel-jonathan** [comment]
This is now fixed. I will be making a release very soon. Thank you!

**Comment by daniel-jonathan** [comment]
Hey, this is now available here [1.2.0](https://github.com/CosmicMind/Graph/releases/tag/1.2.0). Thank you!
